test/IntegrationTest.php

```diff
@@ -1,118 +0,0 @@
-<?php
-/**
- * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
- * @copyright Copyright (c) 2015-2017 Zend Technologies USA Inc. (https://www.zend.com)
- * @license   https://github.com/zendframework/zend-expressive/blob/master/LICENSE.md New BSD License
- */
-
-declare(strict_types=1);
-
-namespace ZendTest\Expressive;
-
-use Fig\Http\Message\StatusCodeInterface as StatusCode;
-use Interop\Http\Server\MiddlewareInterface;
-use Interop\Http\Server\RequestHandlerInterface;
-use PHPUnit\Framework\TestCase;
-use Prophecy\Argument;
-use Psr\Container\ContainerInterface;
-use Psr\Http\Message\ResponseInterface;
-use Psr\Http\Message\ServerRequestInterface;
-use Zend\Diactoros\Response;
-use Zend\Diactoros\Response\EmitterInterface;
-use Zend\Diactoros\ServerRequest;
-use Zend\Expressive\Application;
-use Zend\Expressive\Delegate\NotFoundDelegate;
-use Zend\Expressive\Router\FastRouteRouter;
-
-class IntegrationTest extends TestCase
```

Comment:
Can be reverted?

Reply:
This particular test case is no longer relevant following the refactor.